include/nbl/asset/interchange/IImageAssetHandlerBase.h

```diff
@@ -20,9 +20,9 @@ namespace asset
 
 class IImageAssetHandlerBase : public virtual core::IReferenceCounted
 {
+	core::smart_refctd_ptr<system::ILogger> m_logger;
```

Comment:
no, pass it in the asset loading parameters (same struct where passwords for decryption are)